client/db.go

```diff
@@ -357,15 +357,15 @@ func (db *DB) Run(b *Batch) error {
 	return b.fillResults()
 }
 
-// Tx executes retryable in the context of a distributed transaction. The
+// Txn executes retryable in the context of a distributed transaction. The
 // transaction is automatically aborted if retryable returns any error aside
 // from recoverable internal errors, and is automatically committed
 // otherwise. The retryable function should have no side effects which could
 // cause problems in the event it must be run more than once.
 //
 // TODO(pmattis): Allow transaction options to be specified.
-func (db *DB) Tx(retryable func(tx *Tx) error) error {
-	return newTx(db, 1).exec(retryable)
+func (db *DB) Txn(retryable func(txn *Txn) error) error {
+	return newTxn(*db, 1).exec(retryable)
```

Comment:
I like putting an inline comment `/* depth */` in invocations that have an unguessable parameter.